accessGitHub.js

```diff
@@ -1,13 +1,36 @@
+const EventEmitter = require('events');
 const axios = require('axios');
 const colors = require('colors');
 
+class MyEmitter extends EventEmitter {}
```

Comment:
unless you're actually going to extend this, maybe just create your emitter off the base class.